### ISSUE TITLE
About the Train Seq and Val Seq.

### ISSUE THREAD

**Issue by zkp0113** [Kemo-Huang/JMODT#12]
@Kemo-Huang 
![image](https://user-images.githubusercontent.com/45627441/179188215-e1a14caa-b3c8-4fc7-9c71-7ecb8c66bb24.png)

sorry to bother you.
In the paper, it shows that  The training sequences are split into a training set and a validation set with roughly equal number of frames. Specifically, the training set has 10 sequences and 3975 frames, and the validation set contains 11 sequences and 3945 frames. 
But the code split the dataset into a training set with 10 sequences that contain 3995 frames， and the validation set with 10 sequences that have 3864 frames. 
If the seventeenth sequence were added to the validation set, it would contain 4009 frames. 
![image](https://user-images.githubusercontent.com/45627441/179188281-2a2fbf2e-49f3-48d0-aacd-02c8c79e2b44.png)

So may I have the sequences that were used in your Paper， please？
If you delete some frames when you train or validate please tell me，Thanks a lot.

I check each sequence in the Kitti dataset. Got the number of each Sequence like this.

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:///C:/Users/zkp01/AppData/Local/Temp/msohtmlclip1/01/clip.htm">
<link rel=File-List
href="file:///C:/Users/zkp01/AppData/Local/Temp/msohtmlclip1/01/clip_filelist.xml">
<style>

</style>
</head>

<body link="#0563C1" vlink="#954F72">

Seq-ID | number of    frames
-- | --
0 | 154
1 | 443
2 | 233
3 | 144
4 | 314
5 | 297
6 | 270
7 | 800
8 | 390
9 | 803
10 | 294
11 | 373
12 | 78
13 | 340
14 | 106
15 | 376
16 | 209
17 | 145
18 | 339
19 | 1059
20 | 837

</body>

</html>

**Comment by ShaohuaDong2021** [comment]
Yeah, bro. I got the same number of sequences as your.
